addons/common/functions/fnc_getVehicleIcon.sqf

```diff
@@ -25,7 +25,7 @@ private _objectType = if (_object isEqualType objNull) then {
 } else {
     _object
```

Comment:
if you want to be save case-sensitive wise.

configName (configFile >> "CfgVehicles" >> _object)